feature/more/src/main/java/com/epmedu/animeal/more/account/AccountScreenUI.kt

```diff
@@ -0,0 +1,71 @@
+package com.epmedu.animeal.more.account
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.foundation.button.AnimealSecondaryButtonOutlined
+import com.epmedu.animeal.foundation.button.AnimealTextButtonWithIcon
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.foundation.topbar.BackButton
+import com.epmedu.animeal.foundation.topbar.TopBar
+import com.epmedu.animeal.resources.R
+
+@Composable
+internal fun AccountScreenUI(
+    onBack: () -> Unit
+) {
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
```

Comment:
Please check insets after this PR: https://github.com/AnimealProject/animeal_android/pull/55

I assume we need to add `.statusBarPadding()`

Reply:
Also, please attach video of this feature in PR 🙂

Reply:
Done